### DISCUSSION TITLE
simbot如何添加mirai-console 插件

### DISCUSSION THREAD

**Discussion by bigthouzand** [simple-robot/Discussions#61] [Q&A] [answered]
rt

**Comment by ForteScarlet** [comment, marked as answer]
simbot的mirai组件基于 `mirai-core` 而非 `mirai-console`，不支持添加mirai插件。

**Reply by bigthouzand** [reply]
类似TxCaptchaHelper这种插件，怎么安装？

**Reply by ForteScarlet** [reply]
这与mirai插件无关，这只是一个独立的工具应用，前往其GitHub Releases页面下载文件后安装到手机即可。与mirai相关的问题前往它们的[仓库](https://github.com/mamoe/mirai)或它们的论坛查询。
